YGOProAnalyticsServer/EventHandlers/YgoProAnalysisBasedOnDataFromYgoProServer.cs

```diff
@@ -291,5 +312,27 @@ private void _analyzeBanlist(DuelLog duelLog)
 
             banlistStatistics.IncrementHowManyTimesWasUsed();
         }
+
+        private async Task _addPlayableBanlistsToDecklist(Decklist decklist, IEnumerable<Banlist> banlists)
+        {
+
+            foreach (var banlist in banlists)
+            {
+                if (!decklist.PlayableOnBanlists.Contains(banlist) && _banlistService.CanDeckBeUsedOnGivenBanlist(decklist, banlist))
+                {
+                    decklist.PlayableOnBanlists.Add(banlist);
+                }
+            }
+        }
+
+        private async Task _analyzeCurrentDecklistsForNewBanlists(IEnumerable<Banlist> newBanlists)
+        {
+            if (newBanlists.Count() > 0)
+            {
+                var decklistsFromDb = _db.Decklists.ToList();
```

Comment:
![image](https://user-images.githubusercontent.com/38086920/60061752-a3753700-96f6-11e9-96af-09fe244670a9.png)
Now during one analysis we call _db.Decklists.ToList() twice.  Due to huge amount of decklists in db this is a problem.